README.md

```diff
@@ -177,6 +177,19 @@ Check the Trino connector documentation for more information.
 }}
 ```
 
+#### Seeds
+
+Seeds are CSV files in your dbt project (typically in your data directory), that dbt can load into your data warehouse using the dbt seed command.
+
+For dbt-trino batch_size is defined in macro `trino__get_batch_size()` and default value is `1000`.
+In order to override default value define local macro like:
```

Comment:
In order to override default value define within your project a macro like the following

Reply:
Addressed.